nix/vit-testing/operables.nix

```diff
@@ -0,0 +1,57 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs std;
+  inherit (inputs.cardano-node.packages) cardano-cli;
+  inherit (inputs.cells.jormungandr.packages) jcli;
+  inherit (inputs.cells.catalyst-toolbox.packages) catalyst-toolbox;
+  inherit (inputs.cells.voting-tools-rs.packages) voting-tools-rs;
+
+  l = nixpkgs.lib // builtins;
+
+  mkSimpleOperable = {
+    name,
+    runtimeInputs,
+    args,
+  }: let
+    package = cell.packages.${name};
+  in
+    std.lib.ops.mkOperable {
+      inherit package runtimeInputs;
+      runtimeScript = std.lib.ops.mkOperableScript {
+        inherit args package;
+      };
+    };
+in {
+  registration-service = mkSimpleOperable {
+    name = "registration-service";
+    runtimeInputs = [
+      catalyst-toolbox
+      jcli
+      cardano-cli
+      voting-tools-rs
+    ];
+    args = {
+      "--config" = "/secrets/registration-service.config";
+    };
+  };
+  registration-verify-service = mkSimpleOperable {
+    name = "registration-verify-service";
```

Comment:
I know that you are replicating all services we previously had, but lately registration service and registration-verify-service are deprecated, so i think we don't need those jobs

Reply:
This is good feedback. Now is our chance to update these things, thanks! :)